blocks/core/ff_module/ff_module-task-event-repeater/ff_module-task-event-repeater.rt

```diff
@@ -0,0 +1,11 @@
+<rt-require dependency="../../ff_module/ff_module-task-event/ff_module-task-event.js" as="TaskEvent"/>
+
+<div class="ff_container-task-event-repeater">
+    <ol class="ff_container-task-event-repeater__items">
+        <li class="ff_container-task-event-repeater__item"
+            key= "event-{eventIndex}"
```

Comment:
Is the intent to pass the list of events in the order that they should be rendered? The application currently stores them in chronological order (earliest first), but the designs I've seen shows them in reverse chronological order (latest first), which means that (assuming that this component renders in-order) the key for each event will change each time this component is rendered.